WordPress/src/main/res/layout/fragment_login_no_sites_empty_view.xml

```diff
@@ -0,0 +1,126 @@
+            android:layout_height="wrap_content"
+            android:paddingBottom="@dimen/margin_extra_medium_large"
+            app:layout_constraintBottom_toTopOf="@+id/login_empty_view_illustration"
+            app:layout_constraintStart_toStartOf="@+id/text_displayname"
+            app:layout_constraintTop_toBottomOf="@+id/text_displayname"
+            tools:text="\@droidtester2021" />
+
+        <com.google.android.material.textview.MaterialTextView
+            android:id="@+id/text_displayname"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            app:layout_constraintBottom_toTopOf="@+id/text_username"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/image_avatar"
+            tools:text="Droid Tester"
+            style="@style/Login.EmptyView.TextView.DisplayName"/>
+
+        <ImageView
+            android:id="@+id/login_empty_view_illustration"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:contentDescription="@string/login_no_jetpack_sites"
+            android:gravity="center_horizontal"
+            android:padding="@dimen/margin_extra_large"
+            android:src="@drawable/img_jetpack_empty_state"
+            app:layout_constraintBottom_toTopOf="@+id/login_error_message_title"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/login_empty_view_user_section"/>
+
+        <com.google.android.material.textview.MaterialTextView
+            android:id="@+id/login_error_message_title"
+            style="@style/ActionableEmptyStateTitle"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginBottom="@dimen/margin_medium"
+            android:gravity="center_horizontal"
+            android:lineSpacingExtra="@dimen/login_empty_view_line_spacing"
+            android:text="@string/login_no_jetpack_sites"
+            app:layout_constraintBottom_toTopOf="@+id/login_error_message_text"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/login_empty_view_illustration"/>
+
+        <com.google.android.material.textview.MaterialTextView
+            android:id="@+id/login_error_message_text"
+            style="@style/ActionableEmptyStateSubtitle"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:gravity="center_horizontal"
+            android:lineSpacingExtra="@dimen/login_empty_view_line_spacing"
+            android:text="@string/login_no_jetpack_sites_error_message"
+            android:paddingBottom="@dimen/margin_extra_large"
```

Comment:
1) Question (❓): Why did you use `padding` instead of `margin`?
2) Question (❓): From the design I am seeing that the constraints of this and the above `login_error_message_title` view should fill the parent (left and right). I am seeing that this is not done because of the `ActionableEmptyStateTitle` and `ActionableEmptyStateSubtitle`.  If you remove those you will observe that too. I am actually skeptical about reusing those styles, as those are `Help Activity Styles` it seems. Also, those styles contain `android:layout_xyz` specifics, which make those harder to reuse. Maybe we should start separating all that now and design these styles in a way that will not collide with the current ones or have any `android:layout_xyz` specifications. Wdyt?

Reply:
@ParaskP7 ❓ 
`ActionableEmptyStateTitle` & `ActionableEmptyStateSubtitle` are used in `actionable_empty_view` which are used in empty views across the App. I wanted to keep this view aligned with the other empty views How did you map it to `Help Activity Styles` ?

Reply:
> I wanted to keep this view aligned with the other empty views.

This makes sense. 🙏 

> How did you map it to Help Activity Styles ?

I am seeing this comment on top of these styles, thus my asking about it: `<!--Help Activity Styles-->`

If this group comment is outdated, maybe we should consider replacing it with something better. Wdyt?

Reply:
LOL ... I didn't even see the comment. I used the AS find usages, well because, comments are only good on the day they are written. 😜  But yes, changing the comment in the style file is a good idea.

Reply:
Actually, that comment is only over the `HelpActivitySingleText` and doesn't refer to the ones below it - at least that is how I am reading it. I'm going to leave it as is.

Reply:
> comments are only good on the day they are written.

True! 😝 

> Actually, that comment is only over the HelpActivitySingleText and doesn't refer to the ones below it - at least that is how I am reading it.

You are right, maybe what is missing is another `<!-- Empty State Styles -->` comment to make it more obvious that those styles are not part of the `<!--Help Activity Styles-->`. 🤷